control-plane/roles/gardener-monitoring-certs/tasks/main.yaml

```diff
@@ -0,0 +1,51 @@
+---
+- name: Add ingress certificate for shooted seeds
+  k8s:
+    definition:
+      apiVersion: cert-manager.io/v1
+      kind: Certificate
+      metadata:
+        name: seed-ingress
+        namespace: garden
+        labels:
+          use-clouddns-solver: "true"
```

Comment:
I don't understand this label.

Reply:
Probably it's too opinionated because it assumes Google CloudDNS to be used. In general, it can be used for the `matchLabels` selector in the `ClusterIssuer` solver.